### DISCUSSION TITLE
How to draw lines connecting poles in pole figures

### DISCUSSION THREAD

**Discussion by Elimpie** [mtex-toolbox/mtex#940] [Ask Anything] [answered]
![111PF](https://user-images.githubusercontent.com/82203900/114145656-eed1f700-9916-11eb-8617-e2ebfbacf57d.JPG)
![111PF where I need to draw lines](https://user-images.githubusercontent.com/82203900/114145768-11641000-9917-11eb-908a-290ada1e0d82.JPG)

Good Morning,
I would like to know how to draw lines in MTEX which connect the poles of an orientation in a pole figure as done in some publications. I attached a PF example showing the lines and an example of the pole figure where I would like to draw these lines.
Thanks for your help
Elisa

**Comment by ralfHielscher** [comment]
Hi Elisa,

the command
```
line(vector3d.rand,vector3d.rand)
```
will draw you a line in a spherical projection.

Ralf.

**Reply by Elimpie** [reply]
hello Professor Hielscher,
thanks for the quick reply and the suggestion. I have another question though. How can I draw these lines when I have orientations that I input specifying euler angles (as shown below) or importing a list of euler angles? Should I represent each pole as a vector? is there a specific command in MTEX for this?
![image](https://user-images.githubusercontent.com/82203900/114166112-9d813200-992d-11eb-9233-c7f4cca05a37.png)

Thanks again for the help

**Comment by zmichels** [comment, marked as answer]
Hi,

You will want to make vectors for specificMiller directions from you orientation/s… then use the line() command with those vectors.
One way to do this is as follows:
oV = ori2*Miller(1,1,1,CS)
For each Miller + orientation pair, you will get a vector returned.

**Reply by Elimpie** [reply]
Hello,
Thank you very much. This is exactly the command I needed
Best Regards

**Reply by kilir** [reply]
Just a late comment, for your given problem plotting {1 1 1}, connecting the closest points might be what you are looking for. Obviously, as long as it is not clear where the arbitrary center is, one could just choose the first 4:
```
cs = crystalSymmetry('432')
h= Miller(1,1,1,cs)
o = orientation.rand(1,cs)

plotPDF(o,h)

[h,nsym] = h.symmetrise('unique','antipodal')

% corresponding directions
d = o.*h;

% next one might only want to connect the closest points
pairs = d(nchoosek(1:nsym,2))

[~, id]= sort(angle(pairs(:,1),pairs(:,2)))

% pairs to connect
pairs = pairs(id(1:nsym),:)

hold on
line(pairs(:,1),pairs(:,2),'add2all','antipodal')
hold off
```
Cheers,
Rüdiger

**Reply by Elimpie** [reply]
Hello professor Kilian,
Thanks a lot also for your help! This indeed answer fully to my request.
Best regards
Elisa
